src/ethereum/index.ts

```diff
@@ -450,39 +482,41 @@ export class Ethereum extends Blockchain {
    * Stakes funds into pool.
    *
    * @param address - Sender address.
-   * @param amount - Stake amount in ETH.
+   * @param amount - Stake amount in wei as a BigNumber.
```

Comment:
This change also has to be made for every amount. We need to keep it consistent. Changing to wei makes sense to me considering we use lamports in the Solana module. The other option is to keep it as is.  @Messer4